java-type-checker/java_type_checker/types.py

```diff
@@ -132,27 +135,71 @@ def method_named(self, name):
                 except NoSuchJavaMethod:
                     pass
             raise NoSuchJavaMethod("{0} has no method named {1}".format(self.name, name))
-
+    
+    def is_subtype_of(self, other):
+        is_subtype = False
```

Comment:
My version works, but is much more complicated than yours. I haven't seen the any() Python method before